desktop/cmp/grid/Grid.js

```diff
@@ -85,9 +92,11 @@ export class Grid extends Component {
     }
 
     render() {
-        const {colChooserModel, compact} = this.model,
+        const {colChooserModel, hasSubFields, compact} = this.model,
             {agOptions, showHover, onKeyDown} = this.props,
-            layoutProps = this.getLayoutProps();
+            mobile = XH.app.isMobile,
```

Comment:
consider using all caps 'MOBILE' to help distinguish it?  Sort of orthogonal to all other logic